src/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: 1,
+    title: "Finish homework", 
+  },
+  {
+    id: 2,
+    title: "Go to park",
+  },
+  {
+    id: 3,
+    title: "Cook",
+  },
+];
+  
+function TodoList () {
+  return (
+    <ul>
+      {todoList.map(function(item){
```

Comment:
Very good. 

🎨 **Stylistic suggestion:** I know the text might not be suggesting this yet but it is both common practice and recommend by React to use arrow functions for `map`ping.

```js
{todoList.map( item => {
  return <li key={item.id}>{item.title}</li>
})}
```

Also, since you only have one element you are returning, you could even put this in a single line, using implicit return

```js
{todoList.map( item => <li key={item.id}>{item.title}</li>)}
```